File: problems/algorithms/string-cipher/js/code/cipher.print.mjs
import { rot } from './cipher.mjs';

// Caesar Cipher
// encoding rot2
console.log(rot('abc', 2));
console.log('cde');
console.log(rot('xyz', 2));
console.log('zab');

// encoding rot13
console.log(rot('abc', 13));
console.log('nop');
console.log(rot('xyz', 13));
console.log('klm');

File: src/pages/problems/string-cipher/_codes/ruby/code/print/cipher.print.rb
require_relative "../src/cipher"

# Caesar Cipher
# encoding Cipher.rot2
puts Cipher.rot('abc', 2)
puts 'cde'
puts Cipher.rot('xyz', 2)
puts 'zab'

# encoding Cipher.rot13
puts Cipher.rot('abc', 13)
puts 'nop'
puts Cipher.rot('xyz', 13)
puts 'klm'
